src/fairchem/core/common/distutils.py

```diff
@@ -10,14 +10,16 @@
 import logging
 import os
 import subprocess
-from typing import TypeVar
+from datetime import timedelta
+from typing import Any, TypeVar
 
 import torch
 import torch.distributed as dist
 
 from fairchem.core.common.typing import none_throws
 
 T = TypeVar("T")
+TIMEOUT = timedelta(minutes=30)
```

Comment:
This can now be set from config, defaults to 30 minutes. I dont want to set the default value to high, bc then if could just lead to loosing time when a job is actually hanging and all other ranks are waiting too long for it.